akka-actor-typed/src/main/scala/akka/actor/typed/ActorRefResolver.scala

```diff
@@ -56,6 +55,7 @@ abstract class ActorRefResolver extends Extension {
       ref.path.toSerializationFormatWithAddress(system.address)
 
     ref.toClassic match {
+      case _: IgnoreActorRef => IgnoreActorRef.path.toString // IgnoreActorRef never changes its path
```

Comment:
this will not be handled by classic serialization of such ref

Alt 1)
Special treatment in toSerializationFormatWithAddress, but that is a bit difficult because there we don't easily know that it's an ignoreRef.

Alt 2)
Don't fake the serializationFormat but just use ordinary toSerializationFormatWithAddress? Then when deserializing  we must figure out that it's a ignoreRef.

I'm not sure which alternative has less performance impact on serialization of ordinary refs, which is important.